.NET/Microsoft.Recognizers.Definitions.Common/Korean/NumbersDefinitions.cs

```diff
@@ -112,6 +113,10 @@ public static class NumbersDefinitions
 			'빵',
 			'열'
 		};
+		public static readonly IList<char> TenDirectList = new List<char>
+		{
+			' '

```

Comment:
Why is this not an empty list? Won't it cause problems matching to whitespace in Korean?